pkg/remoteconfig/updater.go

```diff
@@ -57,7 +57,8 @@ type RcServiceConfiguration struct {
 
 // DatadogAgentRemoteConfig contains the struct used to update DatadogAgent object from RemoteConfig
 type DatadogAgentRemoteConfig struct {
-	ID            string                       `json:"name"`
+	ID            string                       `json:""`
```

Comment:
```suggestion
	ID            string                       `json:"id"`
```

Reply:
The id is actually only present inside the metadata of the config, not in the config it self. That's why it will be populated later